app/views/handlers/__init__.py

```diff
@@ -38,3 +39,12 @@ def individual_response_url(
                 journey=journey,
             )
     return None
+
+
+def show_individual_response_link(location, routing_path):
+    if (
+        location.block_id == routing_path[0]
+        and location.list_item_id != "primary-person"
+        and location.list_name == "household"
```

Comment:
In the schema, you can get the section id of the individual section:
```
"individual_response": {
        "for_list": "household",
        "individual_section_id": "individual-section"
    },
```
You then just need to check if the current location is the first block in that section.

Reply:
Removed all this and added `schema.json.get("individual_response", {}).get("individual_section_id")` to search for section id and `schema.get_first_block_id_for_section(section)` to compare with current block. Just need to add some tests for coverage now, thanks!